src/scvmm/azext_scvmm/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
+    "azext.maxCliCoreVersion": "4.0.0"
```

Comment:
Not recommand to add 'azext.maxCliCoreVersion'

Reply:
Removed the maxCliCoreVersion field.